changelog/1.7.0.md

```diff
@@ -1,21 +1,66 @@
 # Scala.meta 1.7.0
 
+## Cross-compilation against Scala.js
+
+It is our great pleasure to announce that the overwhelming majority of scala.meta modules are now supported by Scala.js!
+In 1.7.0, it is possible to do advanced tasks that include parsing, quasiquoting and prettyprinting in a browser.
+The only unsupported modules are `scalahost` and `testkit`.
+
+## Parser for Scaladoc comments
```

Comment:
@xavier-fernandez See the updated PR for a minimalistic description for your amazing contribution! If you'd like to expand it, please submit a follow-up pull request.